dotcom-rendering/src/web/browser/islands/doHydration.tsx

```diff
@@ -1,6 +1,9 @@
 /* eslint-disable @typescript-eslint/no-unsafe-member-access -- necessary for calling our async loaded modules */
+import type { EmotionCache } from '@emotion/react';
+import { CacheProvider } from '@emotion/react';
 import { log } from '@guardian/libs';
-import { h, hydrate, render } from 'preact';
+import { createElement } from 'react';
+import { hydrate, render } from 'react-dom';
```

Comment:
Here I switched from importing directly from `preact` to importing from `react `& `react-dom`. This resolved some type issues we had from trying to use jsx in this file, but shouldn't affect the final build as `react` & `react-dom` and aliased to `preact/compat` in the webpack build. This was the only place in the project we imported directly from preact.